packages/react/src/components/ComposedModal/ComposedModal-test.js

```diff
@@ -179,16 +175,17 @@ describe('ComposedModal', () => {
             </ComposedModal>
           </>
         );
-      };
-
+      }
       render(<ComposedModalExample />);
 
       await userEvent.click(screen.getByText('Click me'), { clickCount: 3 });
-      expect(screen.getByRole('presentation', { hidden: true })).toHaveClass(
-        'is-visible'
-      );
+      await new Promise((resolve) => setTimeout(resolve, 1e3));
+
+      const elementModal = screen.getByRole('presentation', { hidden: true });
+      expect(elementModal).toHaveClass('is-visible');
 
-      expect(screen.getByTestId('test-id-1')).toHaveFocus();
+      const elementInput = screen.getByTestId('test-id-1');
+      expect(elementInput).toHaveFocus();
```

Comment:
is the timeout here necessary if you `await` the `expect` statements?
I.E.: `await expect(elementModal).toHaveClass('is-visible');` instead of `expect(elementModal).toHaveClass('is-visible');`

Reply:
Oh my, sorry… the timeout is a forgotten experiment for fixing. The issue was elsewhere, but this got left behind.
I'll fix it right away